inst/shinydashboard.css

```diff
@@ -58,7 +58,7 @@ a > .info-box {
   color: #333;
 }
 
-/* Disable bouncy transition for sidebar toggle. */
+/* Disable bouncy transition for sidebar toggle.
```

Comment:
because it adresses https://github.com/rstudio/shinydashboard/issues/79 which I included because it made sense to me (at that time at least). Should (Can) I open a new PR for it ?